docs/components/movement-sensor/_index.md

```diff
@@ -43,6 +42,11 @@ Viam also supports generic [sensors](/components/sensor/) and [encoders](/compon
 
 If you are using the micro-RDK, navigate to [Micro-RDK Movement Sensor](/build/micro-rdk/movement-sensor/) for supported model information.
 
+### SparkFun RTK reference station setup
```

Comment:
@npentrel I figured it might be good to add this on the main page so there would be some direction to it-- what do you think about this nesting? Might not be enough info. I could also add similar message directly to component pages?

Reply:
I think it might make more sense on the component pages? It feels a bit odd to me next to the micro rdk section?